.github/workflows/generative-tests.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Long-running generative tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "0 0 * * *"
```

Comment:
Minor post-merge comment: I think it's a good idea to avoid midnight scheduling with cron, exactly because it's the default.  There will be 10,000s of other `schedule` jobs with `0 0 * *` that are synchronised with this one, and it might take longer to run than other times.

In this particular case, I don't think it's a problem at all. But just thought I'd mention it.

Reply:
Thanks, Simon. I've changed this.